tests/all/integration/api/helpers/test_auth.py

```diff
@@ -45,5 +45,17 @@ def test_is_accessible(self):
             logout_user()
             self.assertEqual(auth_manager.is_accessible(), False)
 
+    def test_check_auth_admin(self):
```

Comment:
I think you should also test with admin=False

Reply:
You can create two users and one admin and other non-admin and this should give correct response for each of them

Reply:
@poush Made the requested changes. Please have a look